cmplibrary/src/main/java/com/sourcepoint/cmplibrary/data/ServiceImpl.kt

```diff
@@ -367,50 +366,52 @@ private class ServiceImpl(
         consentActionImpl: ConsentActionImpl,
         env: Env,
         sPConsentsSuccess: ((SPConsents) -> Unit)?,
-        pmId: String?
+        pmId: String? // TODO figure out if we actually need this param, since we DO NOT use it
```

Comment:
@carmelo-iriti it seems like we do not use this param anywhere, do you think we can remove it?

Reply:
@carmelo-iriti I double checked, and neither the `pmId` from the `Service` interface, nor from the `ServiceImpl` is being used

`Service.kt`
![image](https://github.com/SourcePointUSA/android-cmp-app/assets/132656146/12ad6eb1-c129-4f04-ba83-1ff800d819be)

`ServiceImpl.kt`
![image](https://github.com/SourcePointUSA/android-cmp-app/assets/132656146/1c35fb2c-3f18-42a3-ba0a-63ffa3d31d60)